Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/menu/ui/TestMenuFragmentViewModel.kt

```diff
@@ -36,7 +37,8 @@ class TestMenuFragmentViewModel @AssistedInject constructor() : CWAViewModel() {
             PlaygroundFragment.MENU_ITEM,
             DataDonationTestFragment.MENU_ITEM,
             DeltaonboardingFragment.MENU_ITEM,
-            ContactDiaryCommentInfoTestFragment.MENU_ITEM
+            ContactDiaryCommentInfoTestFragment.MENU_ITEM,
+            EventRegistrationTestFragment.MENU_ITEM
```

Comment:
Optional: I would like to add a trailing comma `,` 😄  we use this module frequently to avoid ⬆️ 
I can do that  too in my PR